src/app/config/projectProperties.json

```diff
@@ -1,70 +1,58 @@
 {
     "projectTypes": {
-        "angular": {
-            "displayname": "Office Add-in project using Angular framework",
+        "taskpane": {
```

Comment:
I think we should have these specify the "yo-office" branch, just like for Excel-Custom-Functions.

Reply:
Are you suggesting we create "yo-office" branches for all the projects now?

Reply:
Yes. We don't want merges to master to automatically be deployed into Yo Office.

Reply:
git checkout master
git checkout -b "yo-office"
git push -u origin

Reply:
Yeah that makes sense.  I think we should create yo-office branches now then.  Do you agree?

Reply:
Yes

Reply:
We should also make the "yo-office" branch protected in GitHub.

Reply:
I created yo-office branches and was able to make most of them protected with the exception of the repos that now no longer have a "Settings" link due to the recent removal of a project admin